### DISCUSSION TITLE
FreePlane-Zotero plugin stopped working

### DISCUSSION THREAD

**Discussion by Olifair** [freeplane/freeplane#1743] [Help] [answered]
Hello , 
I hadn't use the Zotero plugin since the last FreePlane update. and now when I right click a node and use "assign citations to node"
I get the following Error :

Error executing Script:
No such property: ui for class: assignCitationsToNode at line -1

using freeplane 1.11.11 and Zotero 6.0.36 and Freeplane- zotero plugin 0.1

Is anybody else observing this ? (anybody else using this plugin ? )

**Comment by macmarrum** [comment, marked as answer]
Hi @Olifair 

The plugin (freeplane-zotero 0.1) works OK for me with Freeplane 1.11.11 and Zotero 6.0.35.

I would try uninstalling freeplane-zotero, restarting Freeplane and installing freeplane-zotero again.

**Comment by HaaiHenkie** [comment]
Hello @Olifair and @macmarrum,

I had a similar problem with a add-on directly after upgrading to v1.11.11

~~~
Error executing Script:
No such property: textUtils for class: GTDSync at line -1
~~~

I solved it by deleting the directories `compiledscripts` and `compiledscripts2` from the Freeplane user directory. Maybe it would be better if these caches are emptied during an upgrade (or downgrade) installation.
